pyproject.toml

```
@@ -12,8 +12,8 @@ authors = [
 ]
 dependencies = [
     "affine",
-    "geopandas>=0.8",
-    "hydromt>=0.9.1, <0.11",
+    "geopandas>=1.0",
+    "hydromt>=0.10, <0.11",
```

I think you can leave the lower version limit unchanged right? It will by default take the latest, but I think everything should work with 0.9 as well. 
```suggestion
    "hydromt>=0.9, <0.11",
```

---

Seems like we have quite some nice fixes in 0.10.0? MOstly related to bugs in raster.py for rotated grids, working with zoomlevels etc

https://github.com/Deltares/hydromt/releases